PassleSync.Core/Services/ConfigService.cs

```diff
@@ -59,15 +59,25 @@ public IEnumerable<string> PassleShortcodes
         {
             get => PassleShortcodesString?.Split(',') ?? Enumerable.Empty<string>(); 
         }
-        public string PostPermalinkPrefix
+        public string PostPermalinkTemplate
         {
-            get => _keyValueService.GetValue("PassleSync.PostPermalinkPrefix");
-            set => _keyValueService.SetValue("PassleSync.PostPermalinkPrefix", value);
+            get => _keyValueService.GetValue("PassleSync.PostPermalinkTemplate");

```

Comment:
We're checking a different property in the key value service but if someone upgrades from an old version of the plugin to a new one this won't exist
we need some code to port the old settings over to the new settings
you can see how this is done in `OptionsService.php` of the WP plugin
same of course applies to people below

Reply:
Resolving this as @klikas pointed out that we don't have anyone using the plugin yet